app/main.py

```
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        com, file_from, new_file = command.split()
+        if com == "cp" and file_from != new_file:
+            with (
+                open(file_from, "r") as file_in,
+                open(new_file, "w") as file_out
+            ):
+                file_out.write(file_in.read())
+        else:
+            pass
+    except Exception:
+        pass
```

Using `except Exception:` is generally considered a bad practice because it catches and hides every possible error, making it very difficult to debug your code. For example, if the source file doesn't exist (`FileNotFoundError`) or you don't have permission to write the destination file (`PermissionError`), this code will silently fail, and you won't know why.

The checklist requires you to "check that the input string contains 3 elements". If `command.split()` doesn't produce exactly three items, it raises a `ValueError`. It would be better practice to specifically catch `ValueError` to handle only the case of a malformed command string.